src/lib/components/TagWidget.svelte

```diff
@@ -76,5 +78,6 @@
 		picture={pictures.find((picture) => picture.tag?.type === widget?.pictureType)}
 		avatar={pictures.find((picture) => picture.tag?.type === 'avatar')}
 		class={className}
+		{titleCase}
```

Comment:
```suggestion
		{titleClassNames}
```

Apply to all widgets